src/Ar/CSVFileLib/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+1.2.4 - Do not create log file if CFG.DisableLogging is true
```

Comment:
Maybe something like: Fix disable logging not preventing log file creation. Is more appropriate